compute/kubernetes/concepts.mdx

```diff
@@ -191,9 +191,9 @@ A Pod is the smallest and simplest unit in the Kubernetes object model. Containe
 <Concept>
 
 ## Pool
-The Pool resource is a group of Scaleway compute instances, organized by type (e.g., GP1-S, GP1-M). It represents the computing power of the cluster and contains the Kubernetes nodes, on which the containers run. Consider the following when creating a Pool:
+The Pool resource is a group of Scaleway Virtual Instances, organized by type (e.g., GP1-S, GP1-M). It represents the computing power of the cluster and contains the Kubernetes nodes, on which the containers run. Consider the following when creating a Pool:
```

Comment:
I thought a while back we agreed to say Instances / Scaleway Instances, and not Virtual Instances? @ldecarvalho-doc @ofranc ?

Reply:
Scaleway Instances I think